setup.py

```diff
@@ -42,7 +42,7 @@
     ],
     install_requires=[
         # from requirements.txt
-        'ccxt>=1.66.32',
+        'ccxt>=1.72.29',
```

Comment:
Please also bump to this or a latter version in `requirements.txt`, otherwise ccxt will be installed twice - first the one from requirements, and then this one (not just in CI, but also in regular installs).